addons/stock_picking_batch/views/stock_picking_batch_views.xml

```diff
@@ -47,23 +49,34 @@
         <field name="model">stock.move.line</field>
         <field name="arch" type="xml">
             <list editable="top" decoration-muted="state == 'cancel'" string="Move Lines" default_order="location_id">
+                <header>
+                    <button class="btn-primary" name="action_put_in_pack" type="object" string="Put in Pack" groups="stock.group_tracking_lot"/>
```

Comment:
doesn't seem to be working. This also isn't working though:

https://github.com/odoo/odoo/blob/e69d7924f60423c2611371ad247a979c94b75cd8/addons/stock/views/stock_move_views.xml#L227-L229

so to be fixed later I guess? :shrug: